ci/authn-gce/get_gce_id_tokens.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/bash
+
+# This script obtains all of the GCE identity tokens required by authn-gce cucumber tests
+# The scripts accepts the GCE instance name as an argument and an optional GCP zone (default is us-central1-a).
+# The script does the following:
+# 1. Validates gcloud is installed (requires a GCP account)
+#   To install gcloud goto: https://cloud.google.com/sdk/docs, download and install the SDK by running:
+#   ./google-cloud-sdk/install.sh
+#   ./google-cloud-sdk/bin/gcloud init
+# 2. Verifies the GCE instance exists and is in status running
+# 3. Executes a ssh curl command and write the output token with the appropriate token name to '../ci/authn-gce/tokens'.
+
+PROGNAME=$(basename $0)
+GCE_ZONE=us-central1-a
```

Comment:
why is this the default zone? why even have a default zone?

Reply:
This is the default zone `gcloud` is using unless you specify otherwise in GCP console settings.

Reply:
Upadte - zone must be specified in gcloud ssh command.
```
ERROR: (gcloud.compute.ssh) Underspecified resource [nessi-gcp-play-1]. Specify the [--zone] flag.
```

Reply:
After some work I was able to work without providing a zone.